packages/reports/src/Components/Form/CompositeCalendar/two-month-picker.tsx

```diff
@@ -1,63 +1,81 @@
-import PropTypes from 'prop-types';
 import moment from 'moment';
 import React from 'react';
 import { Calendar } from '@deriv/components';
 import { addMonths, diffInMonths, epochToMoment, subMonths, toMoment } from '@deriv/shared';
 
-const TwoMonthPicker = React.memo(({ onChange, isPeriodDisabled, value }) => {
-    const [left_pane_date, setLeftPaneDate] = React.useState(subMonths(value, 1).unix());
+type TTwoMonthPicker = {
+    onChange: (date: moment.MomentInput) => void;
+    isPeriodDisabled: (date: moment.MomentInput) => boolean;
+    value: number;
+};
+
+const TwoMonthPicker = React.memo(({ onChange, isPeriodDisabled, value }: TTwoMonthPicker) => {
+    const [left_pane_date, setLeftPaneDate] = React.useState(
+        subMonths(value ? new Date(value * 1000).toISOString() : '', 1).unix()
+    );
     const [right_pane_date, setRightPaneDate] = React.useState(value);
 
-    const navigateFrom = e => {
-        setLeftPaneDate(e.unix());
-        setRightPaneDate(addMonths(e, 1).unix());
+    /**
+     * Navigate from date
+     *
+     * @param {moment.Moment} date
+     */
+    const navigateFrom = (date: moment.Moment) => {
+        setLeftPaneDate(date.unix());
+        setRightPaneDate(addMonths(date.toISOString(), 1).unix());
     };
 
     /**
-     * Only allow previous months to be available to navigate. Disable other periods
+     * Navigate to date
      *
-     * @param date
-     * @param range
-     * @returns {boolean}
+     * @param {moment.Moment} date
      */
-    const validateFromArrows = (date, range) => {
-        return range === 'year' || diffInMonths(epochToMoment(left_pane_date), date) !== -1;
+    const navigateTo = (date: moment.Moment) => {
+        setLeftPaneDate(subMonths(date.toISOString(), 1).unix());
+        setRightPaneDate(toMoment(date).unix());
     };
 
     /**
-     * Validate values to be date_from < date_to
+     * Only allow previous months to be available to navigate. Disable other periods
+     *
+     * @param {moment.Moment} date
+     * @param {Extract<moment.DurationInputArg2, 'month'>} range
      */
-    const shouldDisableDate = date => {
-        return isPeriodDisabled(date.unix());
+    const validateFromArrows = (date: moment.Moment, range: Extract<moment.DurationInputArg2, 'month'>) => {
+        // return range === 'year' || diffInMonths(epochToMoment(left_pane_date), date) !== -1;
```

Comment:
We didn't need this condition? `range === 'year'`

Reply:
> We didn't need this condition? `range === 'year'`

You are right, it's my old habit of commenting everything instead of deleting